packs/ansible/actions/lib/ansible_base.py

```diff
@@ -21,6 +21,19 @@ def __init__(self, args):
         :type args: ``list``
         """
         self.args = args[1:]
+        self._prepend_venv_path()
+
+    @staticmethod
+    def _prepend_venv_path():
+        """
+        Modify PATH env variable by prepending virtualenv path with ansible binaries.
+        This way venv ansible has precedence over globally installed ansible.
+        """
+        venv_path = '/opt/stackstorm/virtualenvs/ansible/bin'
```

Comment:
I was looking for something like that when initially implemented ansible pack, to avoid this `/opt/stackstorm/virtualenvs/ansible/bin` hardcoding string.

But `cfg.CONF.system` is available only if `python-script` is used (part of [sandboxing](https://github.com/StackStorm/st2/blob/master/st2common/st2common/util/sandboxing.py)), not for standalone `local-shell-script`. No?